static/styles/main.css

```diff
@@ -20,10 +20,11 @@ body{
     --gradiant: linear-gradient(to right, #e2336b, #fcac46);
     --background: linear-gradient(180deg, rgb(248, 248, 248) 0%, rgb(255, 255, 255) 100%);
     --lightbackground: linear-gradient( 135deg, #90F7EC 10%, #32CCBC 100%);
+    --pagebackground: #f0f2f5;
 }
 
 .container{
-  width:95%;
+  width:100%;
```

Comment:
Can you add the next line to `about.css` to match these changes?
`width: 80%;` at `.description{... }`

Reply:
Ok